cmd/bot/run.go

```diff
@@ -318,3 +330,14 @@ func queryL2ScannedBlock(db *gorm.DB, cfg *core.Config) (*core.L2ScannedBlock, e
 	}
 	return &l2ScannedBlock, nil
 }
+
+// NOTE: types.L2ContractEvent.FailureReason is defined as `type:varchar(256)` in database.
```

Comment:
why don't just use `text` as the type?

Reply:
Good suggestion.

Both truncating into `varchar(255)` and defining in `text` are good to me. However, `text` seems better. I will change to `text`.